ERCS/erc-7562.md

```diff
@@ -293,6 +293,13 @@ Alt-mempools are served by the same bundlers participating in the canonical memp
 * **[AREP-020]** the alt-mempool becomes THROTTLED based on the [Reputation Calculation](#reputation-calculation)
 * **[AREP-030]** REMOVED
 
+### EIP-7702 Authorizations
+
+* **[AUTH-010]** A UserOperation may only contain a single EIP-7702 authorization tuple.
+* **[AUTH-020]** An account with EIP-7702 delegation can only be used as the Sender of the UserOperation.
+  Using authorized account as any other kind of UserOperation entity is not allowed.
+* **[AUTH-030]** An account with EIP-7702 delegation can only be **accessed** if it is the Sender of the current UserOperation.
```

Comment:
"Accessed" (`*call` or `extcode*`)

Reply:
> "Accessed" (`*call` or `extcode*`)

I think the definition of "accessed" shouldn't be given explicitly in case there are new opcodes, but follow the warm/cold address rules. What do you think?